github_action_report.js

```diff
@@ -0,0 +1,50 @@
+    function myFunction() {
+        var input, filter, table, tr, td, i, txtValue;
+        input = document.getElementById("myInput");
+        filter = input.value.toUpperCase();
+        table = document.getElementById("principalTable");
+        tr = table.getElementsByTagName("tr");
+        for (i = 0; i < tr.length; i++) {
+            td = tr[i].getElementsByTagName("td")[0];
+            if (td) {
+                txtValue = td.textContent || td.innerText;
+                if (txtValue.toUpperCase().indexOf(filter) > -1) {
+                    tr[i].style.display = "";
+                } else {
+                    tr[i].style.display = "none";
+                }
+            }
+        }
+    }
+
+    $(document).ready(function() {
+        $('[data-toggle="tooltip"]').tooltip();
+    });
+
+
+    function toggle(id) {
+        var e = document.getElementById(id)
+        if (e.style.display == 'none')
+            e.style.display = 'table-row-group';
+        else
+            e.style.display = 'none';
+
+    }
+
+    //  Google Analytics
+    (function(i, s, o, g, r, a, m) {
```

Comment:
Please be explicit with the function parameters here

Reply:
Hi Dani, according to : https://developers.google.com/analytics/devguides/collection/analyticsjs , this is how the function is written, do we want to change it in order to be more explicit?